### PR TITLE
.yml Updates

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: rhino.yml [hunk 1]
```diff
@@ -8,59 +8,76 @@ services:
...
         image: seresearch/opendlv-rhino-on-opendlv-on-opendlv-core-on-opendavinci-on-base:v0.1.5
         restart: on-failure
         network_mode: "host"
         volumes:
         - .:/opt/opendlv.data
-        command: "/opt/od4/bin/odsupercomponent --configuration=/opt/opendlv.data/configuration --cid=100"
+        cap_add:
+        - SYS_NICE
+        entrypoint: ""
+        command: "nice -n 0 /opt/od4/bin/odsupercomponent --configuration=/opt/opendlv.data/configuration --cid=110"
```
Comment (chrberger) [inline rhino.yml]: Shouldn't we have `--cid=100` here?

File: rhino.yml [hunk 2]
```diff
@@ -8,59 +8,76 @@ services:
...
+        - CLUON_SHAREDMEMORY_POSIX=1
+        volumes:
+        - /tmp:/tmp
+        cap_add:
+        - SYS_NICE
+        entrypoint: ""
+        command: "nice -n -10 /usr/bin/opendlv-video-x264-encoder --cid=100 --name=video0.i420 --width=640 --height=480"
 
     # Volvo Angle Sensor using lab jack
 #    dev-io-ue9-amd64:
```
Comment (chrberger) [inline rhino.yml:83]: Did we test this?

File: rhino.yml [hunk 3]
```diff
@@ -129,14 +153,17 @@ services:
 #    video-h264-encoder-amd64:
```
Comment (chrberger) [inline rhino.yml:153]: The `opendlv-video-h264-encoder` service does only work when `CLUON_SHAREDMEMORY_POSIX=0` !

File: rhino.yml [hunk 4]
```diff
@@ -129,14 +153,17 @@ services:
...
 #        volumes:
 #        - /tmp:/tmp
-#       command: "--cid=100 --name=video0.i420 --width=640 --height=480"
+#        cap_add:
+#        - SYS_NICE
+#        entrypoint: ""
+#        command: "--cid=100 --name=video0.i420 --width=640 --height=480"
 
     # Encoder for camera-opencv(vp8)
 #    video-vpx-encoder:
```
Comment (chrberger) [inline rhino.yml:171]: The service `opendlv-video-vpx-encoder` must also match with `CLUON_SHAREDMEMORY_POSIX=?` !

File: rhino.yml [hunk 5]
```diff
@@ -158,15 +188,21 @@ services:
...
 #        container_name: dev-gps-nmea
-#        image: chalmersrevere/opendlv-device-gps-nmea-multi:v0.0.9
+#        image: chalmersrevere/opendlv-device-gps-nmea-multi:v0.0.13
 #        restart: on-failure
 #        network_mode: "host"
-#        command: "--nmea_ip=10.42.42.23 --nmea_port=9999 --cid=100"
+#        cap_add:
+#        - SYS_NICE
+#        entrypoint: ""
+#        command: "nice -n -3 /usr/bin/opendlv-device-gps-nmea --nmea_ip=10.42.42.23 --nmea_port=9999 --cid=110"
```
Comment (chrberger) [inline rhino.yml]: Should be a `10.40.40.x` IP

**Review by chrberger — CHANGES_REQUESTED** [review]

File: rhino.yml [hunk 6]
```diff
@@ -8,59 +8,76 @@ services:
...
+        command: "nice -n 0 /opt/od4/bin/odsupercomponent --configuration=/opt/opendlv.data/configuration --cid=110"
 
 ######################################################################################
 #
 # Microservices for RHINO (Volvo FH16).
 #
 ######################################################################################
 
-    # OxTS GPS
+    # OxTS GPS (10.40.40.56)
```
Comment (chrberger) [inline rhino.yml:28]:
```suggestion
    # OxTS GPS (10.40.40.56)
```

File: rhino.yml (hunk 1 above)
Comment (ghost) [inline rhino.yml]:
```suggestion
         command: "nice -n 0 /opt/od4/bin/odsupercomponent --configuration=/opt/opendlv.data/configuration --cid=100"
```

File: rhino.yml (hunk 2 above)
Comment (ghost) [reply rhino.yml:83]: in a way, yes. It's still temporary but it works

File: rhino.yml [hunk 7]
```diff
@@ -129,14 +153,17 @@ services:
...
 #            dockerfile: Dockerfile.amd64
 #        restart: on-failure
 #        network_mode: "host"
 #        ipc: "host"
 #        volumes:
 #        - /tmp:/tmp
-#       command: "--cid=100 --name=video0.i420 --width=640 --height=480"
+#        cap_add:
+#        - SYS_NICE
+#        entrypoint: ""
```
Comment (ghost) [inline rhino.yml:167]:
```suggestion
#       environment:
#       - CLUON_SHAREDMEMORY_POSIX=0
#       entrypoint: ""
```

File: rhino.yml [hunk 8]
```diff
@@ -147,7 +174,10 @@ services:
 #        ipc: "host"
 #        volumes:
 #        - /tmp:/tmp
-#        command: "--cid=100 --name=video0.i420 --width=640 --height=480 --vp8"
+#        cap_add:
+#        - SYS_NICE
+#        entrypoint: ""
+#        command: "nice -n -10 /usr/bin/opendlv-video-vpx-encoder --cid=100 --name=video0.i420 --width=640 --height=480 --vp8"
```
Comment (ghost) [inline rhino.yml:184]:
```suggestion
#        environment:
#        - CLUON_SHAREDMEMORY_POSIX=1
#        command: "nice -n -10 /usr/bin/opendlv-video-vpx-encoder --cid=100 --name=video0.i420 --width=640 --height=480 --vp8"
```

File: rhino.yml (hunk 5 above)
Comment (ghost) [inline rhino.yml]:
```suggestion
#        command: "nice -n -3 /usr/bin/opendlv-device-gps-nmea --nmea_ip=10.40.40.X --nmea_port=9999 --cid=110"    
```

File: rhino.yml (hunk 6 above)
Comment (chrberger) [reply rhino.yml:28]: Fine

File: rhino.yml (hunk 2 above)
Comment (chrberger) [reply rhino.yml:83]: okay

**Review by chrberger — APPROVED** [review]
LGTM

**Pull request merged**